virttest/asset.py

```diff
@@ -376,28 +376,38 @@ def get_file_asset(title, src_path, destination):
     return None
 
 
-def get_asset_info(asset):
+def get_asset_info(asset, dir_=None, section=None):
```

Comment:
Please use a better variable name then `dir_`. Based on the description I'd go with `ini_dir`, `path` or `ini_location`.